pvlib/iotools/psm3.py

```diff
@@ -0,0 +1,133 @@
+"""
+Get PSM3 TMY
+see https://developer.nrel.gov/docs/solar/nsrdb/psm3_data_download/
+"""
+
+import io
+import csv
+import datetime
+import requests
+import pandas as pd
+import pytz
+
+URL = "http://developer.nrel.gov/api/solar/nsrdb_psm3_download.csv"
+
+# 'relative_humidity', 'total_precipitable_water' are note available
+ATTRIBUTES = [
+    'air_temperature', 'dew_point', 'dhi', 'dni', 'ghi', 'surface_albedo',
+    'surface_pressure', 'wind_direction', 'wind_speed']
+
+
+def get_psm3(latitude, longitude, tmy='tmy', interval=60):
+    """
+    Get PSM3 data
+
+    Parameters
+    ----------
+    latitude : float or int
+        in decimal degrees, between -90 and 90, north is positive
+    longitude : float or int
+        in decimal degrees, between -180 and 180, east is positive
+    tmy : str
+        PSM3 parameter "name", see notes below for options, default ``'tmy'``
```

Comment:
numpydoc standard says "Enclose variables in single backticks." 

https://numpydoc.readthedocs.io/en/latest/format.html section 4. Parameters